_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+Legal aspects of renting in Switzerland
+=======================================
+
+This page provides a brief overview of some legal aspects of apartment rental in Switzerland, in the hopes that it will help you navigate them in the future.
+
+## Lease takeovers
+In many countries, one can cancel one’s lease at any date as long as a certain notice period is respected. In Switzerland, this is usually not the case. A typical lease is valid for one year, and tacitly renewed. When you want to leave, you can only do so at its next term, and you also need to inform the landlord 3 months in advance of your intention of moving out. So, if you have moved into an apartment on 21.06.2020, and you want to move out in 2023, you will have to pay rent until 21.06.2023, and your notice should be received by your landlord no later than 21.03.2023 (https://bail.ch/bail/page/faq/10).
+
+Luckily, there is a workaround for that: lease takeovers. If you can find a suitable (i.e. creditworthy) person who is willing to take over your lease at an earlier date, then you will be released from your obligation to honour the contract until its term. In order for that to work, that tenant needs to actually sign the lease. The landlord is free to refuse the new tenant you propose, but if they are unable to provide a legally valid reason (e.g. “this person’s salary is too low for the rent”), then you will be released from the contract. They may try to intimidate you into staying in the apartment until the end of the term by saying that they will refuse anyone you propose, but that is not legal.
```

Comment:
This is not the same thing as a "sublet"/"sublease", right? You could clarify that in the text.